src/main/java/godaddy/registry/jtoolkit2/se/idn/ietf/DomainCheckIeftIdnEPPCommandExtension.java

```diff
@@ -0,0 +1,59 @@
+package godaddy.registry.jtoolkit2.se.idn.ietf;
+import org.w3c.dom.Element;
+import godaddy.registry.jtoolkit2.ErrorPkg;
+import godaddy.registry.jtoolkit2.se.Command;
+import godaddy.registry.jtoolkit2.se.CommandExtension;
+import godaddy.registry.jtoolkit2.se.ExtendedObjectType;
+import godaddy.registry.jtoolkit2.xml.XMLWriter;
+
+/**
+ * <p>Extension for the EPP Domain Check command, representing the Check IDN Domain aspect of the
+ * Internationalized Domain Names extension.</p>
+ *
+ * <p>Use this to set the language tag to be used for an IDN as part of an EPP Domain Check command.
+ * The response expected from a server should be
+ * handled by a Domain Check Response.</p>
+ *
+ * @see godaddy.registry.jtoolkit2.se.DomainCheckCommand
+ * @see godaddy.registry.jtoolkit2.se.DomainCheckResponse
+ * @see <a href="http://godaddyregistry.github.io/doc/idn-1.0/idn-1.0.html">Internationalized Domain Name Extension
+ * Mapping for the Extensible Provisioning Protocol (EPP)</a>
+ */
+public class DomainCheckIeftIdnEPPCommandExtension implements CommandExtension {
```

Comment:
It appears this new class is not much different from `DomainCreateIetfIdnCommandExtension` other than the name. And it's code does not check the passed in Command (e.g. domain create or domain check).

Could you please remove this new class, and
* Rename the existing class from `DomainCreateIetfIdnCommandExtension` to `IetfIdnCommandExtension`?
* Also update the javadoc in the existing class, to remove the emphasis of _Create_, as it is going to work for both _Create_ and _Check_ commands.

Reply:
Done